packages/nc-gui/components/smartsheet/Toolbar.vue

```diff
@@ -2,6 +2,8 @@
 import { IsPublicInj, useSharedView, useSmartsheetStoreOrThrow } from '#imports'
 import ToggleDrawer from '~/components/smartsheet/sidebar/toolbar/ToggleDrawer.vue'
 
+const props = defineProps<{ containerClass?: string }>()
+
```

Comment:
This is an unnecessary prop.
If you bind a class to `Toolbar.vue` it will already be merged automatically with the classes set in this component, meaning
```vue
<Toolbar class="foobar" />
```

will add the class `foobar` to the classes in the component.

So you can just remove this prop and use the regular class attribute to pass any classes you might want to pass.

Reply:
like this?